templator/templator.go

```diff
@@ -157,5 +164,24 @@ func getFileNames(box *packr.Box, dir string) []string {
 		}
 		return nil
 	})
-	return keys
+	return removeTmplDuplicates(keys)
+}
+
+func removeTmplDuplicates(keys []string) []string {
+	filteredKeys := []string{}
+	for _, key := range keys {
+		if !containsStr(keys, key+".tmpl") {
```

Comment:
essentially if we have a `.tmpl` file with the same name, ignore the original file as its only used for local dev